ledger/txtail.go

```diff
@@ -246,8 +253,8 @@ func (t *txTail) committedUpTo(rnd basics.Round) (retRound, lookback basics.Roun
 }
 
 func (t *txTail) prepareCommit(dcc *deferredCommitContext) (err error) {
-	dcc.txTailDeltas = make([][]byte, 0, dcc.offset)
 	t.tailMu.RLock()
+	dcc.txTailDeltas = make([][]byte, 0, dcc.offset)
```

Comment:
Does this move affect anything? since the arguments to the `make` are passed in and the allocation doesn't touch t's state?